lms/envs/common.py

```diff
@@ -617,7 +617,10 @@
         'OPTIONS': {
             'name': 'tracking'
         }
-    }
+    },
+    'adaptive_learning': {
+        'ENGINE': 'track.backends.adaptive_learning.AdaptiveLearningBackend',
```

Comment:
@e-kolpakov It needs to be `ENGINE`, because the [code](https://github.com/open-craft/edx-platform/blob/send-result-events/common/djangoapps/track/tracker.py#L37) that instantiates the different `TRACKING_BACKENDS` [uses that property](https://github.com/open-craft/edx-platform/blob/send-result-events/common/djangoapps/track/tracker.py#L50) to determine the corresponding Python class.